### ISSUE TITLE
rekeying broken in 2.3.0-pre

### ISSUE THREAD

**Issue by wpbrown** [elasticdog/transcrypt#159]
Attempting to rekey always result in an error that repository is already configured.

```
You are about to re-encrypt all encrypted files using new credentials.
Once you do this, their historical diffs will no longer display in plain text.

Proceed with rekey? [y/N] y

transcrypt: the current repository is already configured; see 'transcrypt --display'
```

**Comment by jmurty** [comment]
Hi @wpbrown I haven't been able to reproduce this problem on Mac OS Ventura 13.2.1 or Ubuntu 23.04

Which versions of the following are you using
- Platform and version (e.g. Mac Ventura, Linux Ubuntu, etc)?
- Transcrypt version installed **in the repository**  
  `./.git/crypt/transcrypt --version`
- Path Transcrypt will use for the `openssl` command  
  `git config transcrypt.openssl-path`
- Exact version of the `openssl` command used by Transcrypt  
  E.g. `openssl version`

**Comment by wpbrown** [comment]
@jmurty Sorry about that. It was a late night. I saw "Merge Release 2.2.3" and grabbed the wrong commit hash. This is 2.3.0-pre that is affected (commit `605d9d0`). It looks related to the context changes. 2.2.3 is working.

Steps are simply
```
git clone git@github.com:XXX repo
cd repo
transcrypt -c aes-256-cbc -p XXX
transcrypt -r
```

**Comment by jmurty** [comment]
Hi @wpbrown the rekey bug should now be fixed on `main` as of commit c505199e080f2cd75bf20f96a13c1ad24d6b31bb
